temporal/t.rast.neighbors/t.rast.neighbors.py

```diff
@@ -187,6 +199,16 @@ def main():
             overwrite=overwrite,
             dbif=dbif,
         )
+        semantic_label = map.metadata.get_semantic_label()
+        if new_labels == "input":
+            if semantic_label is not None:
+                new_map.set_semantic_label(semantic_label)
+        elif new_labels == "method":
+            if semantic_label is not None:
+                semantic_label = "%s_%s" % (semantic_label, method)
```

Comment:
You can just use an f-string here. Introduced in Python 3.6. Preferred way of formatting now for simple strings.

```python
semantic_label = f"{semantic_label}_{method}"
```